rules/opentitan_test.bzl

```diff
@@ -152,7 +152,7 @@ def verilator_params(
         "@//hw:verilator",
         "@//hw:fusesoc_ignore",
     ]
-    required_tags = ["verilator", "cpu:4"]
+    required_tags = ["verilator", "cpu:5"]
```

Comment:
This probably ought to depend on the number of threads for which the verilated model is compiled, but it's fine for now.

Reply:
It definitely ought to, but the tags don't like being updated when that information is known to Bazel as far as I can tell so I got stuck trying to implement that.